requirements/local.txt

```diff
@@ -0,0 +1,40 @@
+-r base.txt
+
+ipdb==0.13.9  # https://github.com/gotcha/ipdb
+
+# Email
+django-yubin==1.7.1 # https://github.com/APSL/django-yubin
+
+
+# Testing
+# ------------------------------------------------------------------------------
+mypy==0.931  # https://github.com/python/mypy
+django-stubs==1.9.0  # https://github.com/typeddjango/django-stubs
```

Comment:
These changes for static analysis look great, but may be out of scope for the current PR :-)